Nodejs/Product/Nodejs/TypingsAcquisition.cs

```diff
@@ -28,22 +28,47 @@
 
 namespace Microsoft.NodejsTools {
     internal class TypingsAcquisition {
-        private const string Tsd = "tsd";
-        private const string TsdExe = Tsd + ".cmd";
+        private const string TypingsAcquisitionTool = "typings";
+        private const string TypingsAcquisitionToolExe = TypingsAcquisitionTool + ".cmd";
+        private const string TypingsAcquisitionToolVersion = "1.0.5";
         private const string TypingsDirectoryName = "typings";
 
-        private static SemaphoreSlim tsdGlobalWorkSemaphore = new SemaphoreSlim(1);
+    private static SemaphoreSlim globalWorkSemaphore = new SemaphoreSlim(1);
+
+        /// <summary>
+        /// Path the the private package where the typings acquisition tool is installed.
+        /// </summary>
+        private static string NtvsToolsPath {
+            get {
+                return Path.Combine(
+                    Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData),
+                    "Microsoft",
+                    "Node.js Tools",
+                    "Tools");
+            }
+        }
+
+        /// <summary>
+        /// Full path to the typings acquisition tool.
+        /// </summary>
+        private static string TypingsAcquisitionToolPath {
+            get {
+                return Path.Combine(
+                    NtvsToolsPath,
+                    "node_modules",
+                    ".bin",
+                    TypingsAcquisitionToolExe);
+            }
+        }
```

Comment:
This is something that's going to be constant across versions of NTVS... we don't necessarily need a versioning story now, but it's certainly worth considering.